packages/features/bookings/BookingMultiFiltersStore.ts

```diff
@@ -0,0 +1,52 @@
+import { create } from "zustand";
+
+import type { SVGComponent } from "@calcom/types/SVGComponent";
+import { User, Link } from "@calcom/ui/components/icon";
+
+interface addFilterOption {
+  StartIcon?: SVGComponent;
+  label: "people" | "event_type" | "date_range" | "location";
+}
+
+export type BookingMultiFilterStore = {
+  addFilterOptions: addFilterOption[];
+  toggleOption: (option: addFilterOption) => void;
+  isFilterActive: (label: addFilterOption["label"]) => boolean;
+};
+
+export const useBookingMultiFilterStore = create<BookingMultiFilterStore>((set, get) => ({
```

Comment:
I am not sure if we need to use a zustand store for it. It could have been a local state to FiltersContainer. I am just learning zustand @sean-brydon what do you think?

Reply:
Something simpler will be good I guess as @hariombalhara says.